src/test/java/org/example/P21_StringIntoIntegerTest.java

```diff
@@ -7,9 +7,9 @@
 class P21_StringIntoIntegerTest {
     @Test
     void getNumber() {
-        String text= "1234";
-        int number = P21_StringIntoInteger.stringToInteger(text);
-        assertEquals("1234", text);
+        String number = "1234";
+        int text = P21_StringIntoInteger.stringToInteger(number);
```

Comment:
Naming `text`  an integer number is not good naming. Give variables proper names.